chef/cookbooks/barclamp/libraries/barclamp_library.rb

```diff
@@ -203,6 +204,17 @@ def usage
           self.owner
         end
 
+        def multipath?(device = @device)
+          uuid_path = "/sys/block/#{device}/dm/uuid"
+          File.exist?(uuid_path) && File.open(uuid_path) { |f| f.read(7).start_with?("mpath-") }
+        end
+
+        def multipath_path?
```

Comment:
This is a little hard to understand at first.  Could you add a comment giving an example of a multipath path and its corresponding holder, to clarify that the holder is multipath?